README.md

```diff
@@ -25,9 +25,9 @@ cd interchain-accounts
 make install 
 ```
 
-2. Download and install an IBC relayer. *The following demo is supported by Hermes v0.9.0, and this is currently the recommended relayer.*
+2. Download and install an IBC relayer.
 ```
-cargo install --version 0.9.0 ibc-relayer-cli --bin hermes --locked
+cargo install --version 0.12.0 ibc-relayer-cli --bin hermes --locked
```

Comment:
This should be updated again, once hermes deliver the patch for setting the correct channel ordering on the relay path. Happy path scenarios work fine with `v0.12.0`, issues are only encountered when packets are timing out, but I've left the timeout as int64 max for now.